ark/segmentation/marker_quantification.py

```diff
@@ -293,6 +303,10 @@ def compute_marker_counts(input_images, segmentation_labels, nuclear_counts=Fals
                     marker_counts, regionprops_multi_comp
                 )
 
+                # if regionprops names does not contain multi_comp props then add them
+                if not set(regionprops_multi_comp).issubset(regionprops_names):
```

Comment:
This is so that after the first iteration through the for loop, subsequent calls to `assign_single_compartment_features` will have the right number of columns?

Reply:
Yes. We only want to add  `regionprops_multi_comp` to `regionprops_names` after all of them have been created in `marker_counts`, which happens in the first iteration.